### DISCUSSION TITLE
mydumper in old debians

### DISCUSSION THREAD

**Discussion by aqueos** [mydumper/mydumper#778] [General]
hi,

 it would be very great to have mydumper build also for old debians like debian 8 and 9 those are version where alternative like mysqldump do not exist so it would be great to have a version for those :)

best regards,
Ghislain.

**Comment by davidducos** [comment]
The problem with maintaining old OS is that they are not maintained anymore, so we need to patch our software for them. We don't want to do that. As an alternative you can deploy a VM with a newer OS and take a backup from a remote server.

**Comment by davidducos** [comment]
@aqueos just for you to know and other people that might be interest on it. Now, that dependencies to zlib and zstd has been removed, it is easier to compile on old version. This doesn't mean that we are going to build against old debian, that is not going to happen, what has gone, its gone. However, the good news here, is that we are going to be able to keep the OS that we have from now on.
